test/unit/models/embassy_test.rb

```diff
@@ -1,92 +1,155 @@
-    contact = create(:contact, street_address: 'street-address', country: other_location)
-    office = create(:worldwide_office,
-                    contact: contact,
-                    worldwide_organisation: organisation,
-                    worldwide_office_type: WorldwideOfficeType::EMBASSY_OFFICE_TYPES.first,
-                   )
-    embassy = Embassy.new(location)
-
-    expected = Embassy::RemoteOffice.new(
-      "org-name",
-      other_location,
-      "/world/organisations/org-slug",
-    )
-    assert_equal expected, embassy.remote_office
+  context "when there are organisations with embassy offices in other world locations" do
+    before do
+      organisation = create(:worldwide_organisation,
+                            world_locations: [world_location],
+                            name: "org-name",
+                            slug: "org-slug")
+      @other_location = create(:world_location)
+      contact = create(:contact, street_address: "street-address", country: @other_location)
+      create(:worldwide_office,
+             contact:,
+             worldwide_organisation: organisation,
+             worldwide_office_type: WorldwideOfficeType::EMBASSY_OFFICE_TYPES.first)
+    end
+
+    it "returns true for #can_assist_british_nationals?" do
+      assert embassy.can_assist_british_nationals?
+    end
+
+    it "returns false for #can_assist_in_location?" do
+      assert_not embassy.can_assist_in_location?
+    end
+
+    it "returns #remote_office" do
+      expected = Embassy::RemoteOffice.new(
+        "org-name",
+        @other_location,
+        "/world/organisations/org-slug",
+      )
+      assert_equal expected, embassy.remote_office
+    end
   end
 
-  test '#remote_office returns nil when the world location is not a special case nor does it have all of its offices outside of the world location' do
-    location = create(:world_location, name: 'location-1')
-    organisation = create(:worldwide_organisation,
-                          world_locations: [location],
-                          name: 'org-name',
-                          slug: 'org-slug')
-    contact = create(:contact, street_address: 'street-address', country: location)
-    office = create(:worldwide_office,
-                    contact: contact,
-                    worldwide_organisation: organisation,
-                    worldwide_office_type: WorldwideOfficeType::EMBASSY_OFFICE_TYPES.first,
-                  )
-    embassy = Embassy.new(location)
-
-    assert_nil embassy.remote_office
+  context "when there is an organisation with two embassy offices, one in an unknown location and one in another world location" do
+    before do
+      organisation = create(:worldwide_organisation,
+                            world_locations: [world_location],
+                            name: "org-name",
+                            slug: "org-slug")
+      @other_location = create(:world_location)
```

Comment:
Minor: It might be clearer to use a `let` here given that you've used `let` for other instance vars in the test.

Reply:
Addressed in 7e532085c4 which I'll fixup before merging.